tests/Unit/ParallelAlgorithms/ApparentHorizonFinder/Test_ApparentHorizonFinder.cpp

```diff
@@ -831,49 +901,53 @@ SPECTRE_TEST_CASE("Unit.NumericalAlgorithms.Interpolator.ApparentHorizonFinder",
 
   // Time-dependent tests in distorted frame using both translation and shape
   // maps.
+  const std::optional<std::string> dependency{"FakeVolumeSubfile"};
   tmpl::for_each<tmpl::list<std::true_type, std::false_type>>(
-      [](auto use_shape_map) {
+      [&](auto use_shape_map) {
```

Comment:
Re doxygen: It is broken because the snippet commands are split across several lines.  Doxygen was previously interpreting the empty lambda captures as snippet boundaries, but this PR removed one of them.  (Details seem to be doxygen-version dependent: locally I see the captures suppressing the warning, but not actually causing the include, while the deployed version accepts them fully.)